src/Horarium.Mongo/JobMongoModel.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using Horarium.Repository;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Options;
+
+namespace Horarium.Mongo
+{
+    [MongoEntity("scheduler.jobs")]
```

Comment:
Maybe we should name this collection `horarium.jobs` (and collection for recurrent jobs too)?

Reply:
what about horarium.jobs.scheduler?

Reply:
I think ```horarium.jobs``` is better

Reply:
done